pints/_mcmc/_adaptive.py

```diff
@@ -117,6 +280,13 @@ def run(self):
         # Return generated chain
         return chain
 
+    def chain(self):
```

Comment:
I don't see the added value of this method! It will also hold the chain in memory in a way the user can't do anything about, best remove this!